tests/cellml_files/test_display_name.cellml

```diff
@@ -0,0 +1,682 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We can probably reuse one of the existing test models rather than adding a new one?

Reply:
found one